internal/server/handlers/config.go

```diff
@@ -1,13 +1,24 @@
 package handlers
 
 import (
+	"context"
+	"fmt"
 	"log/slog"
 	"net/http"
 
 	"github.com/aerospike/aerospike-backup-service/v2/pkg/dto"
-	"github.com/aerospike/aerospike-backup-service/v2/pkg/service"
+	"github.com/aerospike/aerospike-backup-service/v2/pkg/model"
 )
 
+func (s *Service) ConfigActionHandler(w http.ResponseWriter, r *http.Request) {
+	switch r.Method {
+	case http.MethodGet:
+		s.readConfig(w)
+	case http.MethodPut:
+		s.updateConfig(w, r)
```

Comment:
Add the default option returning `http.StatusNotFound` or `http.StatusMethodNotAllowed`.